GTBitsOfGood/boggers
# Account recovery

## Account Recovery

Issue Number(s): #23 

Adds ability to reset password through email with token.

### How to Test

1. Go to log in page
2. Type in email
3. Click forgot password
4. Check your email and click on the link in the email body
